File: other/arrgen.py
class Solution:
    def Met(self,n):
        for i in range(1,n+1):
            arr=(0);
            for j in range(0,n-i):
                arr.append(0);
            for k in range(i,0,-1):
                arr.append(k);
        return arr;
if __name__ == "__main__":
    arr=Solution.Met(Solution,4);
    print(arr);

File: other/arrgen.java
class Solution{
    public static int[] Met(int n)
    {
        int[] arr=new int[n*n];
        int f=-1;
        for(int i=1;i<n+1;i++)
        {
            for(int j=0;j<n-i;j++)
            {
                arr[++f]=0;
            }
            for(int k=i;k>0;k--)
            {
                arr[++f]=k;
            }
        }
 
        return(arr);
    }

    public static void main(String args[])
{
    int[] ans=Met(5);
    for(int i=0;i<ans.length;i++)
    System.out.print(ans[i]);
}
}